packages/lib-sourcify/test/verification.spec.ts

```diff
@@ -1,1318 +0,0 @@
-        __dirname,
-        'sources',
-        'FactoryImmutable',
-        'Factory'
-      );
-      const [factoryAddress] = await deployFromAbiAndBytecode(
-        localProvider,
-        factoryFolderPath,
-        signer,
-        [deployValue]
-      );
-
-      // Deploy the child by calling the factory
-      const txReceipt = await callContractMethodWithTx(
-        localProvider,
-        factoryFolderPath,
-        factoryAddress,
-        'deploy',
-        signer,
-        [deployValue]
-      );
-      const childAddress = txReceipt.events.Deployment.returnValues[0];
-      const abiEncoded = localProvider.eth.abi.encodeParameter(
-        'uint',
-        deployValue
-      );
-      const match = await checkAndVerifyDeployed(
-        childFolderPath,
-        sourcifyChainGanache,
-        childAddress,
-        {
-          abiEncodedConstructorArguments: abiEncoded,
-        }
-      );
-
-      expectMatch(match, 'perfect', childAddress);
-    }); */
-
-    /* it('should verify a contract created by a factory contract and has immutables without constructor arguments but with msg.sender assigned immutable', async () => {
-      const childFolderPath = path.join(
-        __dirname,
-        'sources',
-        'FactoryImmutableWithoutConstrArg',
-        'Child'
-      );
-      const factoryFolderPath = path.join(
-        __dirname,
-        'sources',
-        'FactoryImmutableWithoutConstrArg',
-        'Factory'
-      );
-      const [factoryAddress] = await deployFromAbiAndBytecode(
-        localProvider,
-        factoryFolderPath,
-        signer,
-        []
-      );
-
-      // Deploy the child by calling the factory
-      const txReceipt = await callContractMethodWithTx(
-        localProvider,
-        factoryFolderPath,
-        factoryAddress,
-        'createChild',
-        signer,
-        []
-      );
-      const childAddress = txReceipt.events.ChildCreated.returnValues[0];
-      const match = await checkAndVerifyDeployed(
-        childFolderPath,
-        sourcifyChainGanache,
-        childAddress,
-        {
-          msgSender: factoryAddress,
-        }
-      );
-
-      expectMatch(match, 'perfect', childAddress);
-    });
-    */
-    it('should fully verify a contract which is originally compiled and deployed with Unix style End Of Line (EOL) source code, but being verified with Windows style (CRLF) EOL source code', async () => {
-      const contractFolderPath = path.join(
-        __dirname,
-        'sources',
-        'WrongMetadata',
-      );
-      const { contractAddress } = await deployFromAbiAndBytecode(
-        signer,
-        contractFolderPath,
-      );
-
-      const match = await checkAndVerifyDeployed(
-        contractFolderPath,
-        sourcifyChainHardhat,
-        contractAddress,
-      );
-      expectMatch(match, 'perfect', contractAddress);
-    });
-
-    it('should fully verify a contract when a not alphabetically sorted metadata is provided', async () => {
```

Comment:
same question here

Reply:
I don't even know if this is needed anymore since now verification is not based on metadata

Reply:
How about a test that creates a SolidityMetadataContract with non-alphabetically sorted metadata and then tries to verify it?

Reply:
I don't think this is needed because when you import a json file in javascript via 

```
// Get the unsorted metadata
      const metadataPath = path.join(
        __dirname,
        '..',
        'sources',
        'StorageUnsortedMetadata',
        'metadata.json',
      );
      const metadataBuffer = fs.readFileSync(metadataPath);
      const unsortedMetadata = JSON.parse(metadataBuffer.toString());
      ```

it is automatically ordered.

Reply:
Right. This would only make sense if we use a string as input.